08-single-page-web-apps/mongo_mongoose/readme.md

```diff
@@ -374,19 +326,22 @@ http://docs.mongodb.org/manual/core/write-operations-introduction/
 )
 ```
 
-### You do (10 min): Update all restaurants in 20001 to be in DC.
+### You do (10 min):
+
+Update a restaurants to have a new key-value par `{state: "DC"}`
 
 ```
 > db.restaurants.update(
-  {"address.zipcode": 20001},
-  { $set: { state: "DC" }},
-  {multi: true }
+  {"name": "Cookies Corner"},
+  { $set: { state: "DC" }}
```

Comment:
I feel that limiting this change is important.  It is so rare that we want to update *every* document.  But, you skipped the entire discussion of finding (and filtering) documents.